### PR TITLE
Add sample for video ad

### PR DESCRIPTION
**Pull request by alanorozco**
Advanced example that uses the new multiple actions and hide/show features.

Fixes https://github.com/ampproject/amphtml/issues/7840

### CODE REVIEW

File: src/60_AMP_Ads/Video_Ad.html [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+        Watch Video
+      </div>
+      <amp-img src="https://ampbyexample.com/img/tokyo.jpg"
+        width="300"
+        height="168">
+      </amp-img>
+    </div>
+    <div id="video-container"
+         hidden>
+      <amp-video id="my-video"
```
Comment (aghassemi) [inline src/60_AMP_Ads/Video_Ad.html]: I'd put a `loop` on it

Comment (alanorozco) [inline src/60_AMP_Ads/Video_Ad.html]: Done

File: src/60_AMP_Ads/Video_Ad.html [hunk 2]
```diff
@@ -0,0 +1,197 @@
+<!---{
+ "preview": "a4a"
```
Comment (sebastianbenz) [inline src/60_AMP_Ads/Video_Ad.html]: You can specify the ad dimensions for the preview here:

```
<!---{
 "preview": "a4a",
 "width": 300,
 "height": 168
}--->
```

File: src/60_AMP_Ads/Video_Ad.html [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+  </style>
+</head>
+<body>
+
+  <!-- ## Full Markup -->
+  <!--
+  Create the necessary markup to show two layers that represent the two ad
+  states.
+
+  **Splash layer**
```
Comment (sebastianbenz) [inline src/60_AMP_Ads/Video_Ad.html]: You can inline the code section to match the corresponding code blocks, e.g.:

```
<!--  A `<div>` that contains a static image, text, and a button to open the video  layer.-->
<div id="splash-container">
```

The sample doc view will then render the doc followed by a code section including the next html tag and it's content.

Comment (alanorozco) [inline src/60_AMP_Ads/Video_Ad.html]: I did that initially. The example would then be broken since the two divs reference each other by id.

Comment (sebastianbenz) [inline src/60_AMP_Ads/Video_Ad.html]: I see. This should still work if you put the comments above `splash-container` and `video-container`.

Comment (alanorozco) [inline src/60_AMP_Ads/Video_Ad.html]: Not really, because the splash would hide on action and while it would "work" it would look broken" :)

Comment (sebastianbenz) [inline src/60_AMP_Ads/Video_Ad.html]: I actually like it - it demonstrates the behavior of each section in isolation. -> You get the overall behavior in the demo view. You can make this more explicit by saying something like: 'the video layer is initially hidden, but will appear when you click 'watch video' above.'. 

I think it'd be nice to have the sample then split into four sections:

* Introduction
* Setup
* Splash Layer
* Video Layer

But I'm happy either way - pick whatever you think works best. It's your sample 😉.

**Comment by sebastianbenz** [comment]
One other thing I've noticed: can we make the sample behave a bit more ad like and add a link to a landing page? e.g. to https://ampbyexample.com such as in the existing amp ad samples: https://ampbyexample.com/amp_ads/carousel_ad/preview/

File: src/60_AMP_Ads/Video_Ad.html (hunk 3 above)
Comment (alanorozco) [inline src/60_AMP_Ads/Video_Ad.html]: Done. I can change it back if it's deemed confusing :)

**Comment by alanorozco** [comment]
@sebastianbenz ptal

**Comment by sebastianbenz** [comment]
Looks great! One little problem: clicking the watch video button now opens the landing page link as well.

**Comment by alanorozco** [comment]
Thanks for catching that! Fixed.

**Comment by aghassemi** [comment]
:tada:

**Comment by sebastianbenz** [comment]
Live: https://ampbyexample.com/amp_ads/video_ad/preview/ and FYI #714

**Pull request merged**